.prettierrc.yml

```diff
@@ -0,0 +1,12 @@
+---
+tabWidth: 4
```

Comment:
I suggest to do tabWidth 2
Please format all files

Reply:
I changed it to 2